src/tests/zfs/zfs_volume_provisioning/zfs_volume_provisioning_test.go

```diff
@@ -0,0 +1,111 @@
+package zfs_volume_provisioning
+
+import (
+	"testing"
+
+	"github.com/openebs/openebs-e2e/common"
+	"github.com/openebs/openebs-e2e/common/e2e_ginkgo"
+	"github.com/openebs/openebs-e2e/common/zfs"
+
+	"github.com/openebs/openebs-e2e/common/k8stest"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var nodeConfig zfs.ZfsNodesDevicePoolConfig
+var app k8stest.FioApplication
+var err error
+
+func TestLvmVolumeProvisioningTest(t *testing.T) {
```

Comment:
update it Lvm -> ZFS

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/209dfad52b9f563091e7dacdcbd528cb37a5ea53..610dd1547aa5f3e2942fe79ecc6af941787afb4b)